woocommerce/woocommerce-ios
# Order Creation: Release order creation (removes experimental toggle)

Closes: #6536

## Description

This removes the Order Creation experimental toggle, releasing the feature to all users. 🎉 

Note: The top banner on the Orders tab still only mentions simple payments; the banner will be updated in a followup PR.

## Changes

* Removes the order creation section (toggle) from `BetaFeaturesViewController`.
* Removes checks for the experimental feature from `OrdersRootViewController` and `AddOrderCoordinator`, making the orders tab bottom sheet always show both order types (full order creation and simple payments).
* Updates unit and UI tests.

## Testing

Full release test plan: pdcxQM-J5-p2

1. Build and run the app.
2. Go to Menu > Settings > Experimental Features and confirm there is no Order Creation experimental toggle.
3. Go to Orders and tap the + icon. Confirm a bottom sheet appears with both order types (full order creation and simple payments).
4. Select "Create order."
5. Add details to the new order: order status, products, shipping, fee, customer details, and/or customer note.
6. Tap the "Create" button and confirm the order is created and the app navigates to the order details for the new order.

## Screenshots

https://user-images.githubusercontent.com/8658164/160665939-021e0846-84a4-4aea-9025-66bfcdaf96dc.mp4




## Submitter Checklist

Update release notes:
